src/analysis/Environments/Environments.test.ts

```diff
@@ -524,6 +526,46 @@ describe('Environments', () => {
       expect(getTextContentForColumn(fourthAppRow, 8)).toBe(makeCompleteDate(azureApp2.auditInfo.dateAccessed));
     });
 
+    it('Renders Cromwell apps with disabled delete', async () => {
+      // Arrange
+      const props = getEnvironmentsProps();
+
+      const googleApp1 = generateTestAppWithGoogleWorkspace({}, defaultGoogleWorkspace);
+      const azureApp1 = generateTestAppWithAzureWorkspace({ appType: appToolLabels.CROMWELL }, defaultAzureWorkspace);
+      const azureWorkspace2 = generateAzureWorkspace();
+      const azureApp2 = generateTestAppWithAzureWorkspace({ appType: appToolLabels.WORKFLOWS_APP }, azureWorkspace2);
+      asMockedFn(props.leoAppData.listWithoutProject).mockResolvedValue([googleApp1, azureApp1, azureApp2]);
+      asMockedFn(props.useWorkspaces).mockReturnValue({
+        ...defaultUseWorkspacesProps,
+        workspaces: [defaultGoogleWorkspace, defaultAzureWorkspace, azureWorkspace2],
+      });
+      props.permissions = stateAndPermissionsProvider;
+
+      // Act
+      await act(async () => {
+        render(h(Environments, props));
+      });
+
+      // Assert
+      const tableRows: HTMLElement[] = screen.getAllByRole('row').slice(1); // skip header row
```

Comment:
I would assert the number of expected rows first, before making assumptions needed for the other asserts to work.